docs/user/ppl/functions/datetime.rst

```diff
@@ -726,6 +753,31 @@ Example::
     +-------------------------------+
 
 
+UNIX_TIMESTAMP
+--------------
+
+Description
+>>>>>>>>>>>
+
+Usage: Converts given argument to Unix time (seconds since Epoch - very beginning of year 1970). If no argument given, it current Unix time.
+The date argument may be a DATE, DATETIME, or TIMESTAMP string, or a number in YYMMDD, YYMMDDhhmmss, YYYYMMDD, or YYYYMMDDhhmmss format. If the argument includes a time part, it may optionally include a fractional seconds part.
+You can use `FROM_UNIXTIME`_ to do reverse conversion.
+
+Argument type: <NONE>/DOUBLE/DATE/DATETIME/TIMESTAMP
+
+Return type: DOUBLE
+
+Example::
+
+    os> source=people | eval `UNIX_TIMESTAMP(20771122143845)` = UNIX_TIMESTAMP(20771122143845), `UNIX_TIMESTAMP(TIMESTAMP('1996-11-15 17:05:42'))` = UNIX_TIMESTAMP(TIMESTAMP('1996-11-15 17:05:42')) | fields `UNIX_TIMESTAMP(20771122143845)`, `UNIX_TIMESTAMP(TIMESTAMP('1996-11-15 17:05:42'))`
```

Comment:
Why did you use expression as field alias here instead of a simple name like in the integration tests? 
I find this tricky to understand.

Maybe `timeStampFromInteger` and `timeStampFromString`?

Reply:
Thank you, updated in dfd2c1ea.